content/docs/getting-started/user-management/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: User Managment in Backstage
+lastUpdated: '2021-09-21T14:00:00.0Z'
+description: How to manage users in Backstage
+---
+
+## Introduction
+
+Roadie provides backstage with an authentication and user management feature for your new tenant.
+
+When the tenant is created the user requesting the tenant is added as the first user in backstage. This user may choose to add another user.
+
+## User Management
+The user management page is contained under the Administration side bar menu item. ![admin-sidebar.png](admin-sidebar.png)
+
+![user-management-page.png](user-management-page.png)
```

Comment:
Styling of these two look a bit off. Do you think we could just circle the gear on the second photo or something?

https://deploy-preview-389--roadie.netlify.app/docs/getting-started/user-management/

Reply:
Done